### DISCUSSION TITLE
Publish model weights on Github

### DISCUSSION THREAD

**Discussion by MohamedElashri** [lukas-blecher/LaTeX-OCR#49] [General]
For various reasons, Google Drive might not the best way to store the trained model and data. It is not that big and I think GitHub releases of this Repository will be more sufficient for that. Also, Google Drive I/O and download speeds are much smaller in comparison.

**Comment by lukas-blecher** [comment]
I thought about that as well. The reason I did not already do it was that I maybe wanted to provide some binaries in the future once the model is traced. But since that does not seem to be happening any time soon I might upload the model weights here.
Its a good idea. Thanks for the input

**Reply by MohamedElashri** [reply]
Thanks for reply. 

I would also wonder what would be the problem even if you have binaries. You can still add them on github. Actually this would be exactly what releases are designed for.

**Reply by lukas-blecher** [reply]
Done. Thanks for the input
